doc/source/deployment/setup-deployer.rst

```diff
@@ -176,7 +177,3 @@ You can improve SSH connections by enabling pipelining:
    [ssh_connection]
    pipelining = True
    EOF
-
-
-With this done, continue your deployment with the
-:ref:`configuredeployment` page.
```

Comment:
Why was this dropped?

Reply:
It's the only page with such a statement. It's out of sequence with the procedure in the deployment guide. SES Configuration precedes Configure Cloud...Configure Cloud includes Configure for SES Integration, which states that the output of SES Integration must be present.